### PR TITLE
Adding Metadata Application

### PR DESCRIPTION
**Pull request by kegor**
Replicated solution from https://github.com/groupdocs-total/GroupDocs.Total-for-NET-MVC/pull/82.

### CODE REVIEW

File: src/Metadata.aspx.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿using GroupDocs.Total.WebForms.Products.Common.Config;
+using System;
+using System.Web;
+
+namespace GroupDocs.Total.WebForms
+{
+    public partial class Metadata : System.Web.UI.Page
```
Comment (bobkovalex) [inline src/Metadata.aspx.cs]: Codacy Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-WebForms/pullRequest?prid=5096129)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/Products/Metadata/Controllers/MetadataApiController.cs  18
- src/Metadata.aspx.cs  2
- src/Products/Common/Entity/Web/FilePropertyEntity.cs  1
- src/Products/Metadata/Config/MetadataConfiguration.cs  3
- src/client/apps/metadata-e2e/src/plugins/index.js  1
- src/Products/Metadata/Entity/FilePropertyName.cs  1
- src/Products/Common/Util/Comparator/CaseInsensitiveComparer.cs  1
- GroupDocs.Total.WebForms.Test/MetadataTest.cs  1
- src/Products/Metadata/Util/Directory/DirectoryUtils.cs  4
- src/client/apps/metadata-e2e/src/support/app.po.ts  1
         

Clones added
============
- src/Products/Metadata/Controllers/MetadataApiController.cs  20
- src/Metadata.aspx.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-WebForms/pullRequest?prid=5096130&bid=16764458)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- src/client/apps/metadata-e2e/src/plugins/index.js  1
- src/Products/Metadata/Util/Directory/DirectoryUtils.cs  4
- GroupDocs.Total.WebForms.Test/MetadataTest.cs  1
- src/Metadata.aspx.cs  2
- src/Products/Metadata/Controllers/MetadataApiController.cs  18
- src/Products/Metadata/Config/MetadataConfiguration.cs  3
- src/Products/Common/Entity/Web/FilePropertyEntity.cs  1
- src/Products/Metadata/Entity/FilePropertyName.cs  1
- src/client/apps/metadata-e2e/src/support/app.po.ts  1
- src/Products/Common/Util/Comparator/CaseInsensitiveComparer.cs  1
         

Clones added
============
- src/Metadata.aspx.cs  1
- src/Products/Metadata/Controllers/MetadataApiController.cs  20
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-WebForms/pullRequest?prid=5096129&bid=16764457)

File: src/Metadata.aspx.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    {
+        protected void Page_Load(object sender, EventArgs e)
```
Comment (bobkovalex) [inline src/Metadata.aspx.cs]: Codacy Issue found: [Make 'Page_Load' a static method.](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-WebForms/pullRequest?prid=5096129)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/Products/Metadata/Controllers/MetadataApiController.cs  18
- GroupDocs.Total.WebForms.Test/MetadataTest.cs  1
- src/Metadata.aspx.cs  2
- src/Products/Metadata/Util/Directory/DirectoryUtils.cs  4
- src/Products/Metadata/Entity/FilePropertyName.cs  1
- src/client/apps/metadata-e2e/src/support/app.po.ts  1
- src/Products/Common/Util/Comparator/CaseInsensitiveComparer.cs  1
- src/Products/Metadata/Config/MetadataConfiguration.cs  3
- src/client/apps/metadata-e2e/src/plugins/index.js  1
- src/Products/Common/Entity/Web/FilePropertyEntity.cs  1
         

Clones added
============
- src/Products/Metadata/Controllers/MetadataApiController.cs  20
- src/Metadata.aspx.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-WebForms/pullRequest?prid=5096130&bid=16764458)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- src/Products/Common/Entity/Web/FilePropertyEntity.cs  1
- src/Products/Common/Util/Comparator/CaseInsensitiveComparer.cs  1
- src/Metadata.aspx.cs  2
- src/client/apps/metadata-e2e/src/plugins/index.js  1
- src/Products/Metadata/Controllers/MetadataApiController.cs  18
- GroupDocs.Total.WebForms.Test/MetadataTest.cs  1
- src/Products/Metadata/Util/Directory/DirectoryUtils.cs  4
- src/client/apps/metadata-e2e/src/support/app.po.ts  1
- src/Products/Metadata/Entity/FilePropertyName.cs  1
- src/Products/Metadata/Config/MetadataConfiguration.cs  3
         

Clones added
============
- src/Metadata.aspx.cs  1
- src/Products/Metadata/Controllers/MetadataApiController.cs  20
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-WebForms/pullRequest?prid=5096129&bid=16764457)

File: src/Metadata.aspx.cs (hunk 1 above)
Comment (bobkovalex) [inline src/Metadata.aspx.cs]: Codacy Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-WebForms/pullRequest?prid=5096129)

File: src/Metadata.aspx.cs (hunk 2 above)
Comment (bobkovalex) [inline src/Metadata.aspx.cs]: Codacy Issue found: [Make 'Page_Load' a static method.](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-WebForms/pullRequest?prid=5096129)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/client/apps/metadata-e2e/src/support/app.po.ts  1
- src/Products/Metadata/Controllers/MetadataApiController.cs  18
- src/Products/Metadata/Util/Directory/DirectoryUtils.cs  4
- src/Products/Metadata/Entity/FilePropertyName.cs  1
- src/Metadata.aspx.cs  2
- src/Products/Common/Entity/Web/FilePropertyEntity.cs  1
- src/client/apps/metadata-e2e/src/plugins/index.js  1
- src/Products/Metadata/Config/MetadataConfiguration.cs  3
- src/Products/Common/Util/Comparator/CaseInsensitiveComparer.cs  1
- GroupDocs.Total.WebForms.Test/MetadataTest.cs  1
         

Clones added
============
- src/Products/Metadata/Controllers/MetadataApiController.cs  20
- src/Metadata.aspx.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-WebForms/pullRequest?prid=5096130&bid=16764458)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/Metadata.aspx.cs  2
- GroupDocs.Total.WebForms.Test/MetadataTest.cs  1
- src/Products/Metadata/Entity/FilePropertyName.cs  1
- src/Products/Metadata/Controllers/MetadataApiController.cs  18
- src/client/apps/metadata-e2e/src/support/app.po.ts  1
- src/Products/Metadata/Config/MetadataConfiguration.cs  3
- src/client/apps/metadata-e2e/src/plugins/index.js  1
- src/Products/Common/Entity/Web/FilePropertyEntity.cs  1
- src/Products/Common/Util/Comparator/CaseInsensitiveComparer.cs  1
- src/Products/Metadata/Util/Directory/DirectoryUtils.cs  4
         

Clones added
============
- src/Metadata.aspx.cs  1
- src/Products/Metadata/Controllers/MetadataApiController.cs  20
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-WebForms/pullRequest?prid=5096129&bid=16764457)

**Pull request merged**